### ISSUE TITLE
'no results' line for when there is no result in search?

### ISSUE THREAD

**Issue by codebeaker** [getwilds/proof#11]
on the track workflow page, can we have it say 'no results' if there are no results in search (instead of a blank page which makes the user wonder if it's just still thinking)?

**Comment by sckott** [comment]
Yep! Can have a default card that has info on what the user could do to get results, should be straightforward.
